### PR TITLE
chore: Add `command` example

### PR DESCRIPTION
**Pull request by SCasarotto**
This adds an example for the Command component:

Screenshots
image

How to test?

npm run dev
Visit http://localhost:3000/examples/command

Does this PR introduce breaking changes?
Nope, just an example

### CODE REVIEW

File: packages/ariakit/src/command/__examples__/command/style.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    px-4
+    text-base
+    text-primary-2
+    hover:bg-primary-2-hover
+    focus-visible:ariakit-outline
+    aria-disabled:opacity-50
+    dark:bg-primary-2-dark
+    dark:text-primary-2-dark
+    dark:hover:bg-primary-2-dark-hover;
+}
```
Comment (diegohaz) [inline packages/ariakit/src/command/__examples__/command/style.css]: If the styles are the same, we can just import them from the button example:
```suggestion
@import url("../../../button/__examples__/button/style.css");
```

File: packages/ariakit/src/command/__examples__/command/test.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+});
+
+test("tab", async () => {
+  render(<Example />);
+  expect(getByText("Accessible button")).not.toHaveFocus();
+  await press.Tab();
+  expect(getByText("Accessible button")).toHaveFocus();
+});
+
+test("Enter", async () => {
```
Comment (diegohaz) [inline packages/ariakit/src/command/__examples__/command/test.tsx]:
```suggestion
test("enter", async () => {
```

File: packages/ariakit/src/command/__examples__/command/test.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  render(<Example />);
+  await press.Tab();
+  expect(getByText("Accessible button")).toHaveFocus();
+  await press.Enter();
+  expect(alertMock).toHaveBeenCalledTimes(1);
+
+  alertMock.mockRestore();
+});
+
+test("Space", async () => {
```
Comment (diegohaz) [inline packages/ariakit/src/command/__examples__/command/test.tsx]:
```suggestion
test("space", async () => {
```

File: packages/ariakit/src/command/readme.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Checkbox
```
Comment (diegohaz) [inline packages/ariakit/src/command/readme.md]:
```suggestion
# Command
```

**Review by diegohaz — COMMENTED** [review]
Thanks @SCasarotto! I made a few suggestions.

File: packages/ariakit/src/command/__examples__/command/style.css (hunk 1 above)
Comment (SCasarotto) [reply packages/ariakit/src/command/__examples__/command/style.css]: I wasn’t sure about referencing into other components or not. Sounds good. Will fix.

File: packages/ariakit/src/command/readme.md (hunk 4 above)
Comment (SCasarotto) [reply packages/ariakit/src/command/readme.md]: Good catch.

**Comment by SCasarotto** [comment]
I’ll make those changes in a bit. One thing that I didn’t love was the way I selected the component in the tests. GetByText feels brittle, is there a better option or is it fine?

**Comment by diegohaz** [comment]
I'd also use `getByText` in this case. You can also use `getByRole`, but I think it would be a bit verbose for this simple example.

**Review by diegohaz — APPROVED** [review]

**Pull request merged**